source/Playnite.DesktopApp/Controls/SettingsSections/AppearanceAdvanced.xaml

```diff
@@ -8,12 +8,15 @@
              xmlns:p="clr-namespace:Playnite;assembly=Playnite"
              xmlns:sys="clr-namespace:System;assembly=mscorlib"
              xmlns:sm="clr-namespace:System.Windows.Media;assembly=PresentationCore"
+             xmlns:pcmd="clr-namespace:Playnite.Commands;assembly=Playnite" 
              xmlns:pcon="clr-namespace:Playnite.Converters;assembly=Playnite"
              xmlns:pctrls="clr-namespace:Playnite.DesktopApp.Controls"
              mc:Ignorable="d" 
-             d:DesignHeight="500" d:DesignWidth="600">
+             d:DesignHeight="700" d:DesignWidth="600">
     
     <UserControl.Resources>
+        <sys:DateTime x:Key="DateTime">2022-12-22 12:54</sys:DateTime>
+        <pcon:MultiNullableDateToCustomFormatStringConverter x:Key="MultiNullableDateToCustomFormatStringConverter" />

```

Comment:
Unless I'm missing something, this converter is in my opinion unnecessary. I would handle these date examples in code behind in settings view model.

Especially since the converter is only used here, but it's named in very generic way from which it's not clear in what context it can be used.

Reply:
A WIP has been done in https://github.com/JosefNemec/Playnite/pull/3024/commits/8a67067af4f422b61c668fc1b8bf068cccfae211

Only thing missing is setting the strings in the viewmodel to bind and react to changes using the converters here:

https://github.com/JosefNemec/Playnite/blob/8a67067af4f422b61c668fc1b8bf068cccfae211/source/Playnite.DesktopApp/ViewModels/SettingsViewModel.cs#L378-404

I've also left the textblocks to bind in `AppearanceAdvanced.xaml`

If you can do it please go ahead, if not I'll take care of it as soon as possible